nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java

```diff
@@ -172,7 +172,7 @@ public Comment createComment(CoreSession s, Comment comment) {
 
             manageRelatedTextOfTopLevelDocument(session, createdComment);
 
-            notifyEvent(session, CommentEvents.COMMENT_ADDED, documentModel, commentDocModel);
+            notifyEvent(session, CommentEvents.COMMENT_ADDED, commentDocModel);
```

Comment:
don't change the call, since the review on the main PR on comment notification see https://github.com/nuxeo/nuxeo/pull/3623#discussion_r363746037 we decided to keep the call with the four parameters in the case of creation on the comment manager implementation this because we have already the document being commented, [here](https://github.com/nuxeo/nuxeo/blob/master/nuxeo-features/nuxeo-platform-comment/nuxeo-platform-comment/src/main/java/org/nuxeo/ecm/platform/comment/impl/TreeCommentManager.java#L175)